src/deepsparse/evaluation/evaluator.py

```diff
@@ -56,9 +56,10 @@ def evaluate(
 
     # if target is a string, turn it into an appropriate pipeline
     # otherwise assume it is a pipeline
-    pipeline = (
-        create_pipeline(model, engine_type) if isinstance(model, (Path, str)) else model
-    )
+    if isinstance(model, (Path, str)):
+        pipeline, kwargs = create_pipeline(model, engine_type, **kwargs)
```

Comment:
nit: The rules for passing in kwargs here in create_pipeline and later in eval_integration is pretty loose.
Instead of mutating i would have two dicts instead of using one and mutating it

Reply:
@horheynm you are right. For the reasons that we will discuss internally, the design here is not the best and some refactoring will be necessary. But for the sake of landing it for 1.7, i'd ask you to turn a blind eye for now.